src/shared/observability/scope.tsx

```diff
@@ -102,24 +91,9 @@ interface Properties {
   children: ReactNode;
 }
 export const WithObservabilityScope = ({ children }: Properties) => {
-  // injected script doesn't have access to chrome.runtime api so we need to inject extension id into script.id which we set before injecting script into html
-  const extensionId =
-    document.querySelector('[data-testid="idriss-injected-script"]')?.id ?? '';
-
   const scope = useMemo(() => {
-    return createObservabilityScope('injected-app', extensionId);
-  }, [extensionId]);
-
-  useEffect(() => {
-    const captureException = (event: ErrorEvent) => {
-      scope.captureException(event.error);
-    };
-    self.addEventListener('error', captureException);
-
-    return () => {
-      self.removeEventListener('error', captureException);
-    };
-  }, [scope]);
+    return createObservabilityScope('injected-app');
+  }, []);
```

Comment:
this was reporting all errors that were appearing on window object so I think it was captuing errors from outside of our extension